livingdoc-engine/src/main/kotlin/org/livingdoc/engine/execution/examples/decisiontables/DecisionTableFixtureChecker.kt

```diff
@@ -59,8 +65,8 @@ internal object DecisionTableFixtureChecker {
         flatMapper: (T) -> Iterable<String>,
         handler: (String) -> Unit
     ) {
-        elements.forEach {
-            it.getAnnotationsByType(annotation.java)
+        elements.forEach { annotatedElement ->
```

Comment:
`element` might be enough in this context

Reply:
we won't win anything with making it less declarative. No use in making the effort